BrainCloudClient/Assets/BrainCloud/Client/BrainCloud/BrainCloudAppStore.cs

```diff
@@ -294,5 +294,31 @@ public BrainCloudAppStore(BrainCloudClient client)
             ServerCall sc = new ServerCall(ServiceName.AppStore, ServiceOperation.FinalizePurchase, data, callback);
             _client.SendRequest(sc);
         }
+        
+        /// <summary>
+        /// Returns up-to-date eligible 'promotions' for the user and a 'promotionsRefreshed' flag indicating whether the user's promotion info required refreshing
```

Comment:
shouldn't this say: `Refreshes promotions for the player.` ?
Where did you get that doc

Reply:
This was copied straight from the Refresh Promotions API documentation. I'll change it to `Refreshes promotions for the player`.

Reply:
he changed it? because we copied ours from the documentation also. And his make no sense

Reply:
ok you're right. ignore my comment we should change ours. maybe he updated it or we missed that